authlib/oauth2/rfc6749/models.py

```diff
@@ -165,6 +165,30 @@ def get_scope(self):
         """
         raise NotImplementedError()
 
+    def get_code(self):
```

Comment:
Same here, maybe these methods can be part of a dedicated mixin class in the rfc9126 module.